dialect/sql/schema/postgres.go

```diff
@@ -526,3 +526,94 @@ func arrayType(t string) bool {
 	}
 	return true
 }
+
+func (d *Postgres) foreignKeys(ctx context.Context, tx dialect.Tx, tables []*Table) (map[*Table][]*ForeignKey, error) {
+	var fks = make(map[*Table][]*ForeignKey)
+
+	var tableLookup = make(map[string]*Table)
+
+	// TODO: include schema
```

Comment:
```suggestion
	tableLookup := make(map[string]*Table)
```

Why do we return `map[*Table][]*ForeignKey` instead of adding the FKs to the tables?

Reply:
will change it. On my. first pass I wanted the new method to not mutate any of the tables but I will change it